.travis.yml

```diff
@@ -2,7 +2,6 @@ dist: trusty
 language: java
 jdk:
   - openjdk11
-  - oraclejdk8
```

Comment:
Are we sure we want to drop java8 support? The adoptopenjdk will keep supporting it at least until end of 2022